src/ObjectUtils/ElfFile.cpp

```diff
@@ -120,6 +124,15 @@ class ElfFileImpl : public ElfFile {
   std::vector<orbit_grpc_protos::ModuleInfo::ObjectSegment> loadable_segments_;
 };
 
+bool IsHotpatchable(const absl::flat_hash_set<uint64_t>& hotpachable_addresses,
```

Comment:
static?

Reply:
I missed the fact that this class is in an anonymous namespace. Then I'd suggest to move the function as a free function above the class. But as said, no strong opinion.

Reply:
I'll just leave as is; I prefer the have the class definition on top.